apps/web/tsconfig.json

```
@@ -1,5 +1,5 @@
 {
   "extends": "tsconfig/nextjs.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/translations"],
```

```suggestion
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/translations/**/*.js"],
```
I think we can make this more strict to only include the `.js` files while leaving the `.po` files out.